komga/src/main/kotlin/org/gotson/komga/application/tasks/TaskHandler.kt

```diff
@@ -121,6 +123,20 @@ class TaskHandler(
             } ?: logger.warn { "Cannot execute task $task: Book does not exist" }
 
           is Task.RebuildIndex -> searchIndexLifecycle.rebuildIndex()
+
+          is Task.DeleteBook -> {
+            bookRepository.findByIdOrNull(task.bookId)?.let { book ->
+              bookLifecycle.deleteBookFiles(book)
+              taskReceiver.scanLibrary(book.libraryId)
```

Comment:
we should trigger the scan task with the same priority, so it's executed straight away. If not, and there's plenty of tasks in queue already, the scan could get delayed a lot, which would provide a bad UX IMHO.